h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -304,6 +304,9 @@ protected GLMModel deepClone(Key<GLMModel> result) {
     public boolean _generate_scoring_history = false; // if true, will generate scoring history but will slow algo down
     
     public void validate(GLM glm) {
+      if (_remove_collinear_columns && !Solver.IRLSM.equals(_solver))
+        glm.warn("remove_collinear_columns", "remove_collinear_columns only works when IRLSM is " +
```

Comment:
I think it is very sensible to make this a warning on the fix release branch. This way it doesn't break existing scripts.

However, should this be an error in master (so that next major) doesn't let user train such model?

Reply:
@michalkurka I struggled with this one too.  Making it an error may be too destructive and can break a lot of codes.  Let me ask the team.   Wendy